src/Heading.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const header = () => {
+    return (
+      <div>
+        <header className="App-header">CYF Hotel</header>;
+        <img className="Logo-image" src="https://images.unsplash.com/photo-1615880484746-a134be9a6ecf?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8M3x8bW91bnRhaW4lMjBob3RlbHxlbnwwfHwwfHx8MA%3D%3D&auto=format&fit=crop&w=500&q=60"></img>
+      </div>
```

Comment:
Great work, sometimes it is good practice to add 'alt' incase the image doesn't load. [More info here](https://www.w3schools.com/tags/att_img_alt.asp#:~:text=The%20alt%20attribute%20provides%20alternative,user%20uses%20a%20screen%20reader).)